From 301728ffa0c3ce60cc54d28645633a335899cfc8 Mon Sep 17 00:00:00 2001
From: Lon Hohberger <lon@metamorphism.com>
Date: Fri, 30 Apr 2021 16:45:21 -0400
Subject: [PATCH] Set default ssh connect timeout to 30 seconds

In virtualized environments where hosts are shared in a build system,
it seems that a host coming online and being ready to accept commands
within the default of 10 seconds is often too short.  Increase this
to 30 seconds to reduce the number of "timeout waiting for SSH banner
exchange" errors we get while performing indirectionimage builds in
Koji.

Signed-off-by: Lon Hohberger <lon@metamorphism.com>
---
 oz/Linux.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/oz/Linux.py b/oz/Linux.py
index a9e89351..f15260a9 100644
--- a/oz/Linux.py
+++ b/oz/Linux.py
@@ -81,7 +81,7 @@ def get_default_runlevel(self, g_handle):
 
         return runlevel
 
-    def guest_execute_command(self, guestaddr, command, timeout=10):
+    def guest_execute_command(self, guestaddr, command, timeout=30):
         """
         Method to execute a command on the guest and return the output.
         """
